ext/strings_test.go

```diff
@@ -60,6 +60,7 @@ var stringTests = []struct {
 	{expr: `'hello wello'.lastIndexOf('elbo room!!') == -1`},
 	{expr: `'hello wello'.lastIndexOf('elbo room!!!') == -1`},
 	{expr: `'hello wello'.lastIndexOf('hello wello') == 0`},
+	{expr: `'bananananana'.lastIndexOf('nana', 8) == 8`},
```

Comment:
Change it to `'bananananana'.lastIndexOf('nana', 7) == 6`

Reply:
Done.